docs/platform/integration/magento2.md

```diff
@@ -136,8 +137,12 @@ If you're using `demo-v1` this will be set up to use our demo Magento 2 instance
 
 ### Deity Cloud Environment Variables
 
-- `PAYMENT_STRIPE_SECRET_KEY`: your secret key
-- `PAYMENT_STRIPE_PUBLIC_KEY`: your public key
+- `MAGENTO_SITE_URL_WITHOUT_PROTOCOL`: Your Magento site URL (without protocol) e.g. magento.deity.io
```

Comment:
@hobadams could you please also add custom-env-vars to FP repo? currently, we don't have any [env vars for magento](https://github.com/deity-io/falcon-platform/blob/dev/examples/demo-v1/server/config/custom-environment-variables.json), so these vars won't be applied. 

And could we please use `MAGENTO_` prefix for these keys? (as there might be many components/APIs that could be using their own tokens)

Reply:
@Borales - https://github.com/deity-io/falcon-platform/pull/224